<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-02-17T16:56:42 IST" name="Automation" finished-at="2022-02-17T16:57:00 IST" duration-ms="18639">
    <groups>
    </groups>
    <test started-at="2022-02-17T16:56:42 IST" name="SeleniumAutomation" finished-at="2022-02-17T16:57:00 IST" duration-ms="18639">
      <class name="epam.com.SeleniumTestCases.orangeHRMPOM.pages.TestOrangeHRMLoginPage">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:epam.com.SeleniumTestCases.orangeHRMPOM.pages.TestOrangeHRMLoginPage@7d4f9aae]" started-at="2022-02-17T16:56:42 IST" name="setup" finished-at="2022-02-17T16:56:47 IST" duration-ms="5623" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="testLoginEnteringWrongPassword()[pri:0, instance:epam.com.SeleniumTestCases.orangeHRMPOM.pages.TestOrangeHRMLoginPage@7d4f9aae]" started-at="2022-02-17T16:56:47 IST" name="testLoginEnteringWrongPassword" finished-at="2022-02-17T16:56:53 IST" duration-ms="5090" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at epam.com.SeleniumTestCases.orangeHRMPOM.pages.TestOrangeHRMLoginPage.testLoginEnteringWrongPassword(TestOrangeHRMLoginPage.java:43)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLoginEnteringWrongPassword -->
        <test-method signature="testLoginFunctionality()[pri:0, instance:epam.com.SeleniumTestCases.orangeHRMPOM.pages.TestOrangeHRMLoginPage@7d4f9aae]" started-at="2022-02-17T16:56:53 IST" name="testLoginFunctionality" finished-at="2022-02-17T16:56:54 IST" duration-ms="1613" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLoginFunctionality -->
        <test-method signature="testLogingToAnotherBrowser()[pri:0, instance:epam.com.SeleniumTestCases.orangeHRMPOM.pages.TestOrangeHRMLoginPage@7d4f9aae]" started-at="2022-02-17T16:56:54 IST" name="testLogingToAnotherBrowser" finished-at="2022-02-17T16:57:00 IST" duration-ms="6227" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogingToAnotherBrowser -->
      </class> <!-- epam.com.SeleniumTestCases.orangeHRMPOM.pages.TestOrangeHRMLoginPage -->
    </test> <!-- SeleniumAutomation -->
  </suite> <!-- Automation -->
</testng-results>
